pennylane/estimator/templates/select.py

```diff
@@ -443,3 +444,285 @@ def controlled_resource_decomp(
             gate_list.append(GateCount(resource_rep(qre.X), 2 * num_zero_ctrl))
 
         return gate_list
+
+
+class SelectPauli(ResourceOperator):
```

Comment:
The Resources section descriptions for the methods of this class are not always consistent, there are small differences that need to be fixed.